src/guide/kuzzle-backend-setup/aws-marketplace.md

```diff
@@ -4,4 +4,68 @@ algolia: true
 title: AWS Marketplace
 description: AWS Marketplace
 order: 1200
----
\ No newline at end of file
+---
+
+# AWS Marketplace
+> ⚠️ To follow this tutorial, you need a __valid AWS account__.
+
+_In this guide, you'll learn where and how to use our AWS Marketplace AMI. It's a
+ good way to test Kuzzle in a cloud environment.In addtion, we recommend you to
+ use our [Kuzzle Admin Console](http://console.kuzzle.io). It's the most comfy
+ way to play with Kuzzle._
+
+## Get the AMI
+
+Our AMI is stored on AWS Marketplace. It's setup with:
+  * Ubuntu (__16.04__)
+  * Kuzzle (__latest__) with MQTT protocol support.
+  * Elasticsearch (__v5.4.1__).
+  * Redis (__v3.2.12__).
+
+Go to the marketplace and type __kuzzle__ in search form.
+You should see one result:
+
+  ![Search result image]()
```

Comment:
Did you forget the image here?

Reply:
Yeah this PR is WIP. Waiting for AWS Marketplace approval